discordbot/commands/drc.py

```diff
@@ -36,11 +42,15 @@ async def decks(ctx: MtgContext, *, args: str) -> None:
     url = f'{domain}/decks/find?q={q}+output:json_api&page_size={MAX_DECKS_SHOWN}'
 
     try:
-        output = fetch_tools.fetch_json(url)
-    except fetch_tools.FetchException:
+        output = await fetch_tools.fetch_json_async(url)
```

Comment:
It would be nicer to push these into `fetcher` too.

Reply:
i guess, i don't like the duplicated code here. i guess i could add a lower-level `fetcher.dreadrise_json_from_url` and then `fetcher.dreadrise_matchups` and `fetcher.dreadrise_decks` or something?

Reply:
Makes sense to me. In general `fetch_tools` is meant to be low-level plumbing not something you call directly from logic.